opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml

```diff
@@ -354,10 +354,10 @@
          <parameter key="rrd-base-name" value="device-config"/>
          <!-- match service name suffix with config-type -->
          <parameter key="config-type" value="${pattern:configType|requisition:configType|default}"/>
-         <parameter key="username" value="${requisition:dcb:username|admin}"/>
-         <parameter key="password" value="${requisition:dcb:password|admin}"/>
          <parameter key="ssh-port" value="${requisition:ssh-port|22}"/>
          <parameter key="ssh-timeout" value="${requisition:ssh-timeout|60000}"/>
+         <parameter key="username" value="${scv:juniper-vsrx:username|requisition:juniper-vsrx:username|admin}"/>
+         <parameter key="password" value="${scv:juniper-vsrx:password|requisition:juniper-vsrx:password|admin}"/>
```

Comment:
Please let's keep the pristine config as generic as possible.

Reply:
Do we support recursive mate expressions? Like, can I put `${scv:juniper-vsrx:username}` into the value of a metadata `requisition:dcb:username`?

Reply:
Implemented  support for parsing mate expression from the value.  
That's the only way we could overwrite  alias in the requisition.